test/AST-Quake/loop_unroll-1.cpp

```diff
@@ -6,25 +6,29 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: cudaq-quake %s | cudaq-opt --pass-pipeline='builtin.module(expand-measurements,canonicalize,cc-loop-unroll,canonicalize)' | FileCheck %s
+// RUN: cudaq-quake %s | cudaq-opt --pass-pipeline='builtin.module(expand-measurements,canonicalize,cc-loop-unroll,update-register-names,canonicalize)' | FileCheck %s
```

Comment:
should we go ahead and update this pass pipeline to use the `--unrolling-pipeline` pass instead?

Reply:
Yes, I'll change it.

Reply:
Updated in b07ca48a